packages/@dcl/sdk-commands/src/commands/deploy/index.ts

```diff
@@ -124,21 +124,36 @@ export async function main(options: Options) {
   // Uploading data
   const catalyst = await getCatalyst(options.args['--target'], options.args['--target-content'])
 
-  printProgressInfo(options.components.logger, `Uploading data to: ${catalyst.getContentUrl()}...`)
+  let contentURL = options.args['--target-content']
+  if ('fetchAbout' in catalyst) {
+    const about = await catalyst.fetchAbout()
+    printProgressInfo(options.components.logger, `Uploading data to: ${about.content.publicUrl}...`)
+    contentURL = about.content.publicUrl
+  } else {
+    printProgressInfo(options.components.logger, `Uploading data to: ${options.args['--target-content']}...`)
+  }
 
   const deployData = { entityId, files: entityFiles, authChain }
   const position = sceneJson.scene.base
-  const network = chainId === ChainId.ETHEREUM_GOERLI ? 'goerli' : 'mainnet'
+  const network = chainId === ChainId.ETHEREUM_SEPOLIA ? 'sepolia' : 'mainnet'
```

Comment:
In this cases that is easy to support both testnets I would just do so instead of completely removing Goerli

```suggestion
  const network = chainId === ChainId.ETHEREUM_GOERLI ? 'goerli' : chainId === ChainId.ETHEREUM_SEPOLIA ? 'sepolia' : 'mainnet'
```